src/constants/mockData.ts

```diff
@@ -0,0 +1,59 @@
+import { Book } from '@interfaces/book';
+
+export const BOOKS: Book[] = [
```

Comment:
Why are books missing from this list? Check this [JSON](https://docs.google.com/document/d/1vzK9M8Dhh5hdRNQPWpuCa1-vF-EO3sGO971-jm3Kli0)

Reply:
Fixe it.